automated/linux/ltp/skipfile-lkft.yaml

```diff
@@ -18,13 +18,39 @@
 # tests:
 #   List of tests to skip
 
+
+globals:
+# This list represents the devices in the lab from lkft.validation.linaro.org,
+# and the qemu devices available in tuxsuite.
+  - boards: &all_boards
+    - bcm2711-rpi-4-b
+    - dragonboard-410c
+    - dragonboard-845c
+    - e850-96
+    - hi6220-hikey-r2
+    - juno-r2
+    - qcom-qdf2400
+    - qrb5165-rb5
+    - x15
+    - x86
+    - i386
+    - x86_64
+    - qemu_arm
+    - qemu_arm64
+    - qemu_x86_64
+    - qemu_i386
+    - qemu-armv7
+    - qemu-arm64
+    - qemu-x86_64
+    - qemu-i386
+    - fvp-aemva
+
```

Comment:
please add them in alphabetic order and add qemu-riscv64

Reply:
Added and alphabetised :+1: